README.md

```diff
@@ -1,26 +1,86 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Airbyte K8s Operator
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+This is the Kubernetes Python Operator for [Airbyte](https://airbyte.com/).
 
-Use links instead.
--->
+## Description
 
-# airbyte-k8s-operator
+Airbyte is an open-source data integration platform designed to centralize and
+streamline the process of extracting and loading data from various sources into
+data warehouses, lakes, or other destinations.
 
-Charmhub package name: operator-template
-More information: https://charmhub.io/airbyte-k8s-operator
+This operator provides an Airbyte server, and consists of Python scripts which
+wraps the versions distributed by
+[Airbyte](https://hub.docker.com/r/airbyte/server).
 
-Describe your charm in one or two sentences.
+## Usage
```

Comment:
It would be good to have somewhere a list of all of the supporting charms and if they are required or optional. It's not clear if you need Temporal/Minio or if just deploying postgresql will be sufficient.